x11-libs/libX11/libX11-1.7.0.ebuild

```diff
@@ -22,17 +22,18 @@ DEPEND="${RDEPEND}
 	x11-libs/xtrans"
 BDEPEND="test? ( dev-lang/perl )"
 
-pkg_setup() {
-	XORG_CONFIGURE_OPTIONS=(
+src_configure() {
+	local XORG_CONFIGURE_OPTIONS=(
 		$(use_with doc xmlto)
```

Comment:
Do we need to depend on xmlto then?

Reply:
https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/xorg-3.eclass#n227